docs/_includes/extra/script.js

```diff
@@ -2,6 +2,26 @@
 *  doesn't work due to fixed body height when menu is toggled.
 *  This script intercepts clicks on links, toggles the menu off
 *  and performs the anchor navigation. */
+
+window.addEventListener("load", function () {
+    let href = window.location.pathname;
+    const hash = window.location.hash;
+    if (hash!=="") {
+        href=hash
+    }
+    let element = document.querySelector(`a[href="${href}"]`);
+    if (element) {
```

Comment:
will this check work or we need to do `!== undefined` as in old days?

Reply:
querySelector() returns element or null, never return undefined